maintainers/maintainer-list.nix

```diff
@@ -5439,6 +5439,12 @@
     githubId = 424946;
     name = "James Earl Douglas";
   };
+  ear7h = {
```

Comment:
maintainer ear7h (line 5442) should be listed after eamsden, not after earldouglas (line 5436)